dspace-services/pom.xml

```diff
@@ -100,7 +100,7 @@
         <dependency>
             <groupId>net.sf.ehcache</groupId>
             <artifactId>ehcache-core</artifactId>
-            <version>2.4.3</version>
+            <version>2.6.11</version>
```

Comment:
Looks like Travis CI is throwing some unit testing errors related to this upgrade of EhCache. Not sure I understand why. But, we could separate this upgrade out into a separate PR / ticket (if it needs further work/investigation).